src/ott/geometry/costs.py

```diff
@@ -496,13 +498,14 @@ class Bures(CostFn):
 
   Args:
     dimension: Dimensionality of the data.
-    kwargs: Keyword arguments for :func:`~ott.math.matrix_square_root.sqrtm`.
+    sqrtm_kw: Dictionary of keyword arguments to control the
+      behavior of inner calls to :func:`~ott.math.matrix_square_root.sqrtm`.
   """
 
-  def __init__(self, dimension: int, **kwargs: Any):
+  def __init__(self, dimension: int, sqrtm_kw: Dict[str, Any] = None):
```

Comment:
Can you please check the Bures-W tutorial and see if we need to also make changes there?

Reply:
I think I already did? I think we only use Bures explicitly in the barycenter part